requirements.txt

```diff
@@ -14,3 +14,4 @@ faiss-cpu==1.7.4
 sentence-transformers==2.2.2
 langchain-google-genai==0.0.9
 pillow==10.2.0
+langchain_openai==0.1.20
```

Comment:
need to add it also to the conda installation environment_dev.yml

Reply:
included this now